power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/meta_data.hpp

```diff
@@ -104,7 +104,7 @@ struct PGM_MetaAttribute {
 
     template <class StructType, auto member_ptr,
               class ValueType = typename trait_pointer_to_member<decltype(member_ptr)>::value_type>
-    PGM_MetaAttribute(MetaAttributeImpl<StructType, member_ptr>, std::string const& attr_name)
+    PGM_MetaAttribute(MetaAttributeImpl<StructType, member_ptr> /*unused*/, std::string const& attr_name)
```

Comment:
here and all occurrences below: use additional spaces, like `/* unused */` to keep consistency with the rest of the code